pkgs/development/python-modules/moto/default.nix

```
@@ -72,26 +78,184 @@ buildPythonPackage rec {
+      "tests/test_glacier/test_glacier_archives.py"
+      "tests/test_glacier/test_glacier_jobs.py"
+      "tests/test_glacier/test_glacier_vaults.py"
+      "tests/test_iam/test_iam.py"
+      "tests/test_iam/test_iam_cloudformation.py"
+      "tests/test_iam/test_iam_groups.py"
+      "tests/test_iam/test_server.py"
+      "tests/test_iot/test_server.py"
+      "tests/test_iotdata/test_server.py"
+      "tests/test_kinesis/test_kinesis.py"
+      "tests/test_kinesis/test_kinesis_cloudformation.py"
+      "tests/test_kinesis/test_server.py"
+      "tests/test_kinesisvideo/test_server.py"
+      "tests/test_kinesisvideoarchivedmedia/test_server.py"
+      "tests/test_kms/test_kms.py"
+      "tests/test_kms/test_server.py"
+      "tests/test_kms/test_utils.py"
+      "tests/test_logs/test_logs.py"
+      "tests/test_polly/test_server.py"
+      "tests/test_rds/test_rds.py"
+      "tests/test_rds/test_server.py"
+      "tests/test_rds2/test_server.py"
+      "tests/test_redshift/test_redshift.py"
+      "tests/test_redshift/test_server.py"
+      "tests/test_resourcegroupstaggingapi/test_resourcegroupstaggingapi.py"
+      "tests/test_route53/test_route53.py"
+      "tests/test_s3/test_s3.py"
+      "tests/test_s3/test_s3_cloudformation.py"
+      "tests/test_s3/test_s3_lifecycle.py"
+      "tests/test_s3/test_s3_storageclass.py"
+      "tests/test_s3/test_s3_utils.py"
+      "tests/test_s3bucket_path/test_s3bucket_path.py"
+      "tests/test_s3bucket_path/test_s3bucket_path_combo.py"
+      "tests/test_secretsmanager/test_server.py"
+      "tests/test_ses/test_server.py"
+      "tests/test_ses/test_ses.py"
+      "tests/test_ses/test_ses_boto3.py"
+      "tests/test_ses/test_ses_sns_boto3.py"
+      "tests/test_sns/test_application.py"
+      "tests/test_sns/test_application_boto3.py"
+      "tests/test_sns/test_publishing.py"
+      "tests/test_sns/test_publishing_boto3.py"
+      "tests/test_sns/test_server.py"
+      "tests/test_sns/test_subscriptions.py"
+      "tests/test_sns/test_subscriptions_boto3.py"
+      "tests/test_sns/test_topics.py"
+      "tests/test_sns/test_topics_boto3.py"
+      "tests/test_sqs/test_server.py"
+      "tests/test_sqs/test_sqs.py"
+      "tests/test_ssm/test_ssm_boto3.py"
+      "tests/test_ssm/test_ssm_docs.py"
+      "tests/test_sts/test_server.py"
+      "tests/test_sts/test_sts.py"
+      "tests/test_swf/models/test_activity_task.py"
+      "tests/test_swf/models/test_decision_task.py"
+      "tests/test_swf/models/test_timeout.py"
+      "tests/test_swf/models/test_workflow_execution.py"
+      "tests/test_swf/responses/test_activity_tasks.py"
+      "tests/test_swf/responses/test_activity_types.py"
+      "tests/test_swf/responses/test_decision_tasks.py"
+      "tests/test_swf/responses/test_domains.py"
+      "tests/test_swf/responses/test_timeouts.py"
+      "tests/test_swf/responses/test_workflow_executions.py"
+      "tests/test_swf/responses/test_workflow_types.py"
```

```suggestion
      "tests/test_swf/models/"
      "tests/test_swf/responses/"
```

Did you test if we could combine this a bit? Over a 100 lines for disabled tests is quite excessive.

---

This would require https://github.com/NixOS/nixpkgs/pull/113167

---

Maybe we should just merge this without using disabledTestFiles/disabledTestPaths for now (just with pytestFlagsArray of ["--ignore A" "--ignore B" ...]) and then sort it out again when the dust settles?

---

Yeah you are right.

---

Okay, I reverted this PR to use the --ignore pytestFlagsArray design, and added comments pointing to the relevant PRs and a note to fix.